src/client/login/widgets/TextButton.tsx

```diff
@@ -16,47 +21,40 @@ const useStyles = makeStyles((theme) =>
         backgroundColor: 'transparent',
       },
     },
+    tag: {
+      display: 'contents',
+    }
   }),
 )
 
-const TextButton = ({
+const TextButton: FunctionComponent<TextButtonProps> = ({
   children,
   className,
   href,
-  target,
   disabled,
   onClick,
-}) => {
+}: PropsWithChildren<TextButtonProps>) => {
   const classes = useStyles()
   return (
-    <Button
-      className={classNames(className, classes.textButton)}
-      href={href}
-      target={target}
-      disabled={disabled}
-      onClick={onClick}
-      variant="text"
-      size="large"
-      disableFocusRipple
-      disableRipple
-    >
-      {children}
-    </Button>
+    <a className={classes.tag} href={href} target="_blank" rel="noopener noreferrer">
```

Comment:
any particular reason why you're changing the structure here?

Reply:
The button from material ui does not have a 'target' as a prop. - https://material-ui.com/api/button/

Wrapping the button with anchor tag is a work around.